api/v1/swagger.yaml

```diff
@@ -6950,10 +6950,10 @@ definitions:
       #   description: >
       #    The number of direct children subjects under this subject.
       createdBy:
+        type: string
         readOnly: true
-        $ref: "#/definitions/UsersResponse"
         description: >
-          User who created this Aspect.
+          Id of the User who created this Aspect.
```

Comment:
since subject is also modified in this pull request, the type is added to the subject response.
changes to the lens response will be included shortly.